File: MySQL.java
import java.sql.*;

public class MySQL {
    public void MySQLconnect(){
        try {
            String Username = "s434804";
            String Password = "*******";
            String URL = "jdbc:mysql://mysql.wmi.amu.edu.pl/s434804_Szkola_Muzyczna";
            Class.forName("com.mysql.jdbc.Driver");
            Connection c = DriverManager.getConnection(URL,Username,Password);
            Statement s = c.createStatement();
            ResultSet r = s.executeQuery("SELECT * FROM SzkolaMuzyczna");

            while(r.next())
            {
                System.out.println("Imie: " + r.getString(1));
                System.out.println("Nazwisko: " + r.getString(2));
                System.out.println("Instrument: " + r.getString(3) + "\n");
            }

        }catch(Exception exc) {
            System.out.println("Error: " + exc);
        }
    }
}

File: MySQL.cs
using System;
using System.Data;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using MySql.Data.MySqlClient;

namespace DBController
{
    class MySQL
    {
        public static void Main(string[] args)
        {
            string connect = "Server=mysql.wmi.amu.edu.pl;Database=s434804_Szkola_Muzyczna;Uid=s434804;Pwd=<haslo>;SslMode=none";
            MySqlConnection con = new MySqlConnection(connect);

            con.Open();
            MySqlCommand command = con.CreateCommand();
            command.CommandText = "SELECT Imie,Nazwisko,Instrument FROM SzkolaMuzyczna";

            IDataReader reader = command.ExecuteReader();
            Console.WriteLine("IMIE" + " " + "NAZWISKO" + " " + "INSTRUMENT");
            while (reader.Read())
            {
                string name = (string)reader["Imie"];
                string surname = (string)reader["Nazwisko"];
                string instrument = (string)reader["Instrument"];
                Console.WriteLine(name + " " + surname + " " + instrument);
            }
            reader.Close();
            command.Dispose();
            con.Close();
        }
    }
}
